### PR TITLE
feat: Backend for Notify

### PR DESCRIPTION
**Pull request by Hridhima**

### CODE REVIEW

File: Backend/models/User.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        type: String,
+        required: true,
+        unique: true,
+     },
+
+     password: {
+        type: String,
+        required: true,
+     },
+     date: {
```
Comment (himanshu007-creator) [inline Backend/models/User.js:20]: date is fine, but its should be more meaningful in terms as in created_at or updated_at

File: Backend/routes/auth.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
+const express = require('express');
+const { query, validationResult, body } = require('express-validator');
+const router = express.Router();
+const User = require('../models/User');
+const bcrypt = require('bcryptjs');
+var jwt = require('jsonwebtoken');
+const fetchuser = require('../middleware/fetchUser');
+
+const JWT_SECRET = "Hridhima$react"
```
Comment (himanshu007-creator) [inline Backend/routes/auth.js:9]: Kindly use .env file, dont hardcode these secrets

File: Backend/package.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "inotes-backend",
+  "version": "1.0.0",
+  "description": "Your Notes on the Cloud",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "serve": "nodemon index.js "
+  },
+  "author": "",
```
Comment (himanshu007-creator) [inline Backend/package.json]: kindly update authors
[
    {"name":"Himanshu","url": "https://github.com/himanshu007-creator",}
    {"name":"Hridhima","url":"https://github.com/Hridhima"}
  ]

**Review by himanshu007-creator — CHANGES_REQUESTED** [review]
Kindly check copmments in `package.json`

File: Backend/models/User.js (hunk 1 above)
Comment (Hridhima) [reply Backend/models/User.js:20]: got it.
Thankyou !

File: Backend/routes/auth.js (hunk 2 above)
Comment (Hridhima) [reply Backend/routes/auth.js:9]: got it
thankyou

File: Backend/package.json (hunk 3 above)
Comment (himanshu007-creator) [reply Backend/package.json]: Thanks for incorporating these changes @Hridhima  😄

**Review by himanshu007-creator — APPROVED** [review]
LGTM 🚀

**Pull request merged**